modin/conftest.py

```diff
@@ -610,8 +609,9 @@ def s3_base(worker_id, monkeysession):
         # to do that is to use the `worker_id`, which is unique, to determine what port to point
         # to. We arbitrarily assign `5` as a worker id to the master worker, since we need a number
         # for each worker, and we never run tests with more than `pytest -n 4`.
```

Comment:
this comment now needs to be either removed or fixed, as it is no longer correct 🙃

Reply:
Fixed